UefiTestingPkg/AuditTests/PagingAudit/UEFI/Dxe/App/AArch64/DxePagingAuditTests.c

```diff
@@ -104,8 +104,6 @@ NoReadWriteExecute (
                 if (!CanRegionBeRWX (Address, SIZE_4KB)) {
                   UT_LOG_ERROR ("Memory Range 0x%llx-0x%llx is Read/Write/Execute\n", Address, Address + SIZE_4KB);
                   FoundRWXAddress = TRUE;
-                } else {
-                  UT_LOG_WARNING ("Memory Range 0x%llx-0x%llx is Read/Write/Execute. This range is excepted from the test.\n", Address, Address + SIZE_4KB);

```

Comment:
Do you want to print to the serial port? I am not sure how much printouts that would be though.

Reply:
As it is, it could print hundreds of lines to the serial port if even one 2MB exempt region has been split.

In a future update to this app, I may just merge contiguous exempt regions to a single print in the XML. For now, this is fine.